presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -25,6 +25,9 @@ services:
   presto-master:
     extends:
       service: java-8-base
+    #image: 'teradatalabs/centos6-java8-oracle-mapr:${DOCKER_IMAGES_VERSION}'
+    # Will replace the image once we have that on docker-hub
```

Comment:
nvm the 'parameterize the image comment'. This file should not be changed at all for the mapr integration. Instead, there should be a separate `mapr.yml` and a new profile, called `mapr` whose `compose.sh` would include the `mapr.yml` similarly to how all other `compose.sh` work. The image should be overridden in mapr.yml.